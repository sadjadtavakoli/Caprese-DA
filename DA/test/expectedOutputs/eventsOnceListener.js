
#4 function  listner1-test/unit_tests/eventsOnceListener.js-4-6 entered throught event connection1 emitted by function arrowFunction-test/unit_tests/eventsOnceListener.js-1-12
#4 function listner1-test/unit_tests/eventsOnceListener.js-4-6 exited to function arrowFunction-test/unit_tests/eventsOnceListener.js-1-12
#end Execution