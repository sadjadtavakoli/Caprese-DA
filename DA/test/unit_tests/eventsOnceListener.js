var events = require('events');
var eventEmitter = new events.EventEmitter();

function listner1() {
    //   do nothing 
}
eventEmitter.once('connection1', listner1);
eventEmitter.addListener('connection2', listner1);

eventEmitter.emit('connection1');
eventEmitter.emit('connection1');
eventEmitter.emit('connection1');