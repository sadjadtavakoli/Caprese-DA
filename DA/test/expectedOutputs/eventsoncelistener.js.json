{"adder-test/unit_tests/eventsoncelistener.js-8-10":{"impacted":["t_1","t_12"]},"adder2-test/unit_tests/eventsoncelistener.js-11-13":{"impacted":["t_1"]},"emitter-test/unit_tests/eventsoncelistener.js-15-19":{"impacted":["t_1","t_4","t_7"]},"keyMap":{"t_1":"eventsoncelistener.js-test/unit_tests/eventsoncelistener.js-1-24","t_4":"adder-test/unit_tests/eventsoncelistener.js-8-10","t_7":"listner1-test/unit_tests/eventsoncelistener.js-4-6","t_8":"adder2-test/unit_tests/eventsoncelistener.js-11-13","t_12":"emitter-test/unit_tests/eventsoncelistener.js-15-19"}}